dependent/processing-and-application-of-ceramics.csl

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded">
+  <info>
+    <title>Processing and Application of Ceramics</title>
+    <id>http://www.zotero.org/styles/processing-and-application-of-ceramics</id>
+    <link href="http://www.zotero.org/styles/processing-and-application-of-ceramics" rel="self"/>
+    <link href="http://www.zotero.org/styles/frontiers-in-optics" rel="template"/>
+    <link href="http://www.tf.uns.ac.rs/publikacije/PAC/index.html" rel="documentation"/>
```

Comment:
Can you change the documentation link to this? 
http://www.tf.uns.ac.rs/publikacije/PAC/guideforauthors.html